storybooknative/src/InputField/index.js

```diff
@@ -94,14 +115,23 @@ const InputField = (props: Props) => {
     required,
     ...rest
   } = props;
+  const isFilled = !!props.value;
   return (
     <React.Fragment>
-      {label !== undefined && label}
+      {label && !inlineLabel && <FormLabel label={label} isFilled={isFilled} required={required} />}
       <StyledInputContainer size={size} disabled={disabled} error={error}>
+        {label &&
+          inlineLabel && (
+            <StyledInlineLabel size={size} filled={isFilled} disabled={disabled}>
+              {required && <StyledAsterisk filled={isFilled}>* </StyledAsterisk>}
```

Comment:
Maybe <StyledAsterisk could be exported with the `*` so we don't have to fill it every time?

```
Const StyledAsteriskWrapper = styled(Text, {...});
export StyledAsterisk = props => <StyledAsteriskWrapper {...props}>* </StyledAsteriskWrapper>
```